gitbot.py

```diff
@@ -731,6 +731,7 @@ def _validate_commit(
         'Move',
         'Refactor',
         'Remove',
+        'Rename',
```

Comment:
You should update your git commit settings such that they show your real name and email address:

```
git config user.name "Your Name"
git config user.email "Your rackspace email address"
```

Once you've done that and amended your commit, then I'll merge your PR.

Reply:
@guyincognito I think I fixed it?

Reply:
@csgodd17 wrote:
> @guyincognito I think I fixed it?

The Github UI is terrible for viewing commits, but after cloning your forked repo and checking out your branch, the commit author and committer look good to me.  I'll go ahead and merge this.